src/Evolution/Systems/RelativisticEuler/Tags.hpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+class DataVector;
+
+namespace RelativisticEuler {
+namespace Tags {
+
+/// The rest-mass density of the fluid
+template <typename DataType = DataVector>
```

Comment:
Same question about why these need to be templates.

Reply:
Same answer as before :)